gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/mapper/PageMapper.java

```diff
@@ -100,6 +102,22 @@ public interface PageMapper {
     @Mapping(target = "uploadDate", source = "createdAt")
     MediaEntity mapMedia(Media media);
 
+    @Mapping(target = "type", source = "type")
+    @Mapping(target = "configuration", qualifiedByName = "serializeJsonConfiguration")
+    PageSource mapSourceConfigurationToPageSource(SourceConfiguration sourceConfiguration);
+
+    @Named("serializeJsonConfiguration")
```

Comment:
Seeing as you import the `ConfigurationSerializationMapper`, you are able to use this method named [here](https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/mapper/ConfigurationSerializationMapper.java#L37).

So this named method in `PageMapper` is actually a duplicate and can be deleted 😅

Reply:
Removed